next-tavla/src/Admin/types/organizations.ts

```diff
@@ -9,3 +9,13 @@ export type TOrganizationsColumn = keyof typeof OrganizationsColumns
 export const ORGANIZATIONS_COLUMNS = Object.keys(
     OrganizationsColumns,
 ) as TOrganizationsColumn[]
+
+export const MemberTableColumn = {
+    email: 'E-post',
+    actions: 'Valg',
+}
+
+export type TMemberTableColumn = keyof typeof MemberTableColumn
+export const MEMBER_TABLE_COLUMNS = Object.keys(
+    MemberTableColumn,
+) as TMemberTableColumn[]
```

Comment:
Syntes ikke vi trenger å definere typer for dette. La oss heller lage de to feltene manuelt i table headeren